index.html

```diff
@@ -0,0 +1,256 @@
+
+            <span class="about">Groove to the infectious tunes while knowing that your ticket purchase is helping to pave the way for a brighter future. Proceeds from this unforgettable concert tour will directly benefit our community's education initiatives, providing vital resources, scholarships, and opportunities for students of all ages. Every move you make on the dance floor contributes to building a stronger foundation for learning and growth.</span>
+            <svg viewBox="0 0 100 100" class="dial">
+              <use xlink:href="sprite.svg#dial"></use>
+            </svg>
+            <svg viewBox="0 0 100 100" class="record-vinyl about">
+              <use xlink:href="sprite.svg#record-vinyl"></use>
+            </svg> 
+          </p>
+          <h3 class="dates home">26.08(SAT) ~ 30.12(SAT)</h3>
+          <h5 class="home">@ Arsenal Studio, Ivan de Bodout Colisseum, and more.</h5>
+          <h5 class="about">Are you vendor that would like to partner with our events?
+            <span>Contact us via email:</span>
+            <span class="email">reggaeducaton2023@mail.com</span>
+          </h5>
+          
+        </article>
+        <svg viewBox="0 0 100 100" class="checkers">
+          <use xlink:href="sprite.svg#checkers"></use>
+        </svg>
+        <svg viewBox="0 0 100 100" class="headset">
+          <use  xlink:href="sprite.svg#headset"></use>
+        </svg>
+        <svg viewBox="0 0 100 100" class="wave-square-1 home">
+          <use xlink:href="sprite.svg#wave-square"></use>
+        </svg>
+        <svg viewBox="0 0 100 100" class="wave-square-2 home">
+          <use xlink:href="sprite.svg#wave-square"></use>
+        </svg>
+        <svg viewBox="0 0 100 100" class="music-alt home">
+          <use  xlink:href="sprite.svg#music-alt"></use>
+        </svg>
+      </section>
+      <section class="program home">
+        <h3 class="sectionHeading">Main Program</h3>
+        <div class="badges">
+          <div class="badge">
+            <svg viewBox="0 0 100 100" class="music-alt">
+            <use  xlink:href="sprite.svg#road"></use>
+            </svg>
+            <h6>La Calle</h6>
+            <p>Our master of ceremonies Evie Queen takes to the street, through the history of reggaeton and introduces our featured artist</p>
+          </div>
+          <div class="badge">
+            <svg viewBox="0 0 100 100" class="music-alt">
+            <use  xlink:href="sprite.svg#route"></use>
+            </svg>
+            <h6>The Journey</h6>
+            <p>The featured artist tells the story of their journey and place in reggaeton's history</p>
+          </div>
+          <div class="badge">
+            <svg viewBox="0 0 100 100" class="music-alt">
+            <use  xlink:href="sprite.svg#q_n_a"></use>
+            </svg>
+            <h6>Q & A</h6>
+            <p>The artist will take questions from the audience.</p>
```

Comment:
- [ ] In the mobile version of your application, I suggest that you increase the size of the text. This will help the people with poor sight to easily interact with the content of your webpage.

Your design 😎 
![image](https://github.com/Drod86/reggaeducaton/assets/99394374/519a0034-fd0d-4a66-b095-da620112c04a)

original design 😎 
![image](https://github.com/Drod86/reggaeducaton/assets/99394374/30aef434-6c32-4905-b5f7-89e159a772b0)